Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -131,8 +132,9 @@ class JitBaseBlockCache
   // is valid (MSR.IR and MSR.DR, the address translation bits).
   static constexpr u32 JIT_CACHE_MSR_MASK = 0x30;
 
-  static constexpr u32 FAST_BLOCK_MAP_ELEMENTS = 0x10000;
-  static constexpr u32 FAST_BLOCK_MAP_MASK = FAST_BLOCK_MAP_ELEMENTS - 1;
+  static constexpr u64 FAST_BLOCK_MAP_SIZE = 0x2'0000'0000;
```

Comment:
You should document how one arrives at this number. You have a guest memory space of 4 GB and PPC instructions are always aligned to 4 bytes, so that gives you 4 GB / 4 bytes = 0x4000'0000 possible entries in the map (I guess it's a lookup table now technically but whatever). Each entry points to a `JitBlock*`, which is 8 bytes on 64 bit machines. That's 8 GB, or 0x2'0000'0000 bytes.